### PR TITLE
New package: yubioath-desktop-4.2.0

### PR DESCRIPTION
**Pull request by ColdPhoenix00**

### CODE REVIEW

**Comment by the-maldridge** [comment]
The update files should not be necessary.  Please remove them.

**Comment by ColdPhoenix00** [comment]
@the-maldridge Done.

File: srcpkgs/python3-pyotherside/template [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+maintainer="ColdPhoenix <coldphoenix@interia.pl>"
+license="ISC"
+homepage="https://thp.io/2011/pyotherside"
+distfiles="https://github.com/thp/pyotherside/archive/${version}.tar.gz"
+checksum=00049d5f42cac448368bc2a521edb8de36bb6d2a624e195b7f1004236758b805
+
+if [ -n "$CROSS_BUILD" ]; then
+	hostmakedepends+=" qt5-host-tools qt5-declarative-devel qt5-svg-devel"
+	PATH="$PATH:${XBPS_CROSS_BASE}/usr/bin"
+	CFLAGS="-I=/usr/include/python3.6m"
```
Comment (Vaelatern) [inline srcpkgs/python3-pyotherside/template]: which of course breaks on updating our python to 3.7

Comment (ColdPhoenix00) [reply srcpkgs/python3-pyotherside/template]: `python3-pyotherside` doesn't like being cross compiled, so I had to force python include path. As a workaround, I can suggest disabling cross compiling completely.

Comment (ColdPhoenix00) [reply srcpkgs/python3-pyotherside/template]: I've set it to nocross if you like that better.

**Pull request closed without merging**